ui/src/containers/CreateClient.js

```diff
@@ -1,217 +1,162 @@
 import React, { Component } from 'react';
-import { Grid, Wizard, Button } from 'patternfly-react';
 import { connect } from 'react-redux';
-import { get as _get } from 'lodash-es';
+import { createApp } from '../actions/apps';
+
+import {
+  FormGroup,
+  ControlLabel,
+  Button, Modal, Alert, Icon
+} from 'patternfly-react';
 import PlatformItem from '../components/create_client/PlatformItem';
-import CreateAndroidClient from '../components/create_client/CreateAndroidClient';
-import CreateCordovaClient from '../components/create_client/CreateCordovaClient';
-import CreateIOSClient from '../components/create_client/CreateIOSClient';
-import CreateXamarinClient from '../components/create_client/CreateXamarinClient';
-import { ResultsLine } from '../components/create_client/ResultsLine';
+import PlatformItems from '../components/create_client/PlatformItems';
+import '../components/create_client/create_client.css';
 import {
   PLATFORM_ANDROID,
   PLATFORM_IOS,
   PLATFORM_CORDOVA,
   PLATFORM_XAMARIN,
   CREATE_CLIENT_TYPE,
-  CLIENT_CONFIGURATION,
-  CREATE_CLIENT_APP_ID,
-  CREATE_CLIENT_NAME,
-  WIZARD_SELECT_PLATFORM,
-  WIZARD_CREATION_RESULT,
-  WIZARD_CONFIGURE_CLIENT,
 } from '../components/create_client/Constants';
-import { createApp } from '../actions/apps';
-import '../components/create_client/create_client.css';
+import CreateAndroidClient from '../components/create_client/CreateAndroidClient';
+import CreateCordovaClient from '../components/create_client/CreateCordovaClient';
+import CreateXamarinClient from '../components/create_client/CreateXamarinClient';
+import CreateIOSClient from '../components/create_client/CreateIOSClient';
```

Comment:
it's likely that we need to make those platforms configurable. If it's not too much trouble, can we make them became part of props of the component, so that we can easily change them, ideally in just 1 place (the top level view)